### PR TITLE
21.2.4+

### PR DESCRIPTION
**Pull request by fshpsmgc**

### CODE REVIEW

File: CS/WebSite/Web.Config [hunk 1]
```diff
@@ -1,4 +1,3 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- 
     Note: As an alternative to hand editing this file you can use the
```
Comment (dirk-pieterse) [inline CS/WebSite/Web.Config:2]:
```suggestion
    Note: You can use the web admin tool to configure settings for your application instead
```

File: Readme.md [hunk 2]
```diff
@@ -3,21 +3,83 @@
...
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# A general technique of using cascading ASPxComboBoxes
+# Combo Box for ASP.NET Web Forms - How to Implement Cascading Combo Boxes
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e2355/)**
 <!-- run online end -->
 
+This demo shows how to use two [ASPxComboBox](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxComboBox) editors to implement cascading combo boxes. In the demo, a selection in the first combo box (Country) filters the item list of the second combo box (City).
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
This demo shows how to use two [ASPxComboBox](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxComboBox) editors to implement cascading combo boxes. In the demo, the selection in the first combo box (Country) filters the item list of the second combo box (City).
```

File: Readme.md [hunk 3]
```diff
@@ -3,21 +3,83 @@
...
+
+![example demo](demo.gif)
+
+Use the following technique to setup a cascade of the [ASPxComboBox](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxComboBox) editors:
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
Use the following technique to setup a cascade of [ASPxComboBox](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxComboBox) editors:  
```

File: Readme.md [hunk 4]
```diff
@@ -3,21 +3,83 @@
...
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e2355/)**
 <!-- run online end -->
 
+This demo shows how to use two [ASPxComboBox](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxComboBox) editors to implement cascading combo boxes. In the demo, a selection in the first combo box (Country) filters the item list of the second combo box (City).
+
+![example demo](demo.gif)
+
+Use the following technique to setup a cascade of the [ASPxComboBox](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxComboBox) editors:  
+**Respond to a selection change** of the first combo box (in its client [SelectedIndexChanged](https://docs.devexpress.com/AspNet/js-ASPxClientComboBox.SelectedIndexChanged) event) and **initiate a callback request** for the second combo box to **filter data source items** on the server (use a combination of the client [PerformCallback](https://docs.devexpress.com/AspNet/js-ASPxClientCallback.PerformCallback(parameter)) method and the server [Callback](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxCallback.Callback) event).
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
**Respond to a selection change** of the first combo box (in its client [SelectedIndexChanged](https://docs.devexpress.com/AspNet/js-ASPxClientComboBox.SelectedIndexChanged) event) and **initiate a callback request** for the second combo box to **filter data source items** on the server (use a combination of the client [PerformCallback](https://docs.devexpress.com/AspNet/js-ASPxClientCallback.PerformCallback(parameter)) method and server [Callback](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxCallback.Callback) event).
```

**Review by dirk-pieterse — APPROVED** [review]

File: CS/WebSite/Web.Config (hunk 1 above)
Comment (fshpsmgc) [reply CS/WebSite/Web.Config:2]: Web.Config is an auto-generated file, so I don't think it was necessary to review its contents. I probably should've specified the files to review in the Trello card.

**Pull request merged**